src/spec.c

```diff
@@ -1333,6 +1353,30 @@ void IndexSpec_Free(IndexSpec *spec) {
 
 //---------------------------------------------------------------------------------------------
 
+// Assumes this is called from the main thread with no competing threads
+// Also assumes that the spec is existing in the global dictionary, so
+// we use the global reference as our guard and access the spec dierctly.
+// This function consumes the Strong reference it gets
+void IndexSpec_RemoveFromGlobals(StrongRef ref) {
+  IndexSpec *spec = StrongRef_Get(ref);
+
+  // Remove spec from global index list
```

Comment:
isn't it the rm?

Reply:
it is, but semantically the comment is true